.github/workflows/ci.yml

```diff
@@ -17,8 +17,14 @@ jobs:
       - uses: actions/checkout@v4.2.2
       - name: Setup Cue
         uses: cue-lang/setup-cue@a93fa358375740cd8b0078f76355512b9208acb1 # v1.0.1
+        with:
+          version: "v0.13.0-alpha.4"
```

Comment:
I've specified this alpha version as its the only release that contains the fix https://github.com/cue-lang/cue/commit/93c1421c23ac8d5ddc8910a9186f5b94e5252ea9